tests/OutputFormatter/JUnitOutputFormatterTest.php

```diff
@@ -0,0 +1,124 @@
+use SensioLabs\Deptrac\Dependency\Result;
+use SensioLabs\Deptrac\DependencyContext;
+use SensioLabs\Deptrac\DependencyResult\Dependency;
+use SensioLabs\Deptrac\DependencyResult\InheritDependency;
+use SensioLabs\Deptrac\OutputFormatter\ConsoleOutputFormatter;
+use SensioLabs\Deptrac\OutputFormatter\JUnitOutputFormatter;
+use SensioLabs\Deptrac\OutputFormatter\OutputFormatterInput;
+use SensioLabs\Deptrac\RulesetEngine\RulesetViolation;
+use Symfony\Component\Console\Output\BufferedOutput;
+
+class JUnitOutputFormatterTest extends TestCase
+{
+    public function testGetName()
+    {
+        $this->assertEquals('junit', (new JUnitOutputFormatter())->getName());
+    }
+
+    public function basicDataProvider()
+    {
+        yield [
+            [
+                'LayerA',
+                'LayerB'
+            ],
+            [
+                new RulesetViolation(
+                    new InheritDependency(
+                        'ClassA',
+                        'ClassB',
+                        new Dependency('OriginalA', 12, 'OriginalB'),
+                        new FlattenAstInherit(
+                            AstInherit::newExtends('ClassInheritA', 3), [
+                                AstInherit::newExtends('ClassInheritB', 4),
+                                AstInherit::newExtends('ClassInheritC', 5),
+                                AstInherit::newExtends('ClassInheritD', 6),
+                            ]
+                        )
+                    ),
+                    'LayerA',
+                    'LayerB'
+                ),
+            ],
+            '
+                JUnitReportdump:<?xmlversion="1.0"encoding="UTF-8"?><testsuites><testsuiteid="1"package=""name="LayerA"hostname="localhost"tests="1"failures="1"errors="0"time="0"><testcasename="LayerA-ClassA"classname="ClassA"time="0"><failuremessage="ClassA:0mustnotdependonClassB(LayerAonLayerB)"type="WARNING"/></testcase></testsuite></testsuites>
+            ',
+        ];
+
+        yield [
+            [
+                'LayerA',
+                'LayerB'
+            ],
+            [
+                new RulesetViolation(
+                    new Dependency('OriginalA', 12, 'OriginalB'),
+                    'LayerA',
+                    'LayerB'
+                ),
+            ],
+            'JUnitReportdump:<?xmlversion="1.0"encoding="UTF-8"?><testsuites><testsuiteid="1"package=""name="LayerA"hostname="localhost"tests="1"failures="1"errors="0"time="0"><testcasename="LayerA-OriginalA"classname="OriginalA"time="0"><failuremessage="OriginalA:12mustnotdependonOriginalB(LayerAonLayerB)"type="WARNING"/></testcase></testsuite></testsuites>',
+        ];
+
+        yield [
+            [
+
+            ],
+            [
+
+            ],
+            'JUnitReportdump:<?xmlversion="1.0"encoding="UTF-8"?><testsuites/>',
+        ];
+    }
+
+    /**
+     * @param array $layers
+     * @param array $violations
+     * @param       $expectedOutput
```

Comment:
the `@param` tags are not really necessary

Reply:
Will remove them.